dash-renderer/src/components/core/DocumentTitle.react.js

```diff
@@ -29,8 +31,10 @@ class DocumentTitle extends Component {
 
 DocumentTitle.propTypes = {
     isLoading: PropTypes.bool.isRequired,
+    update_title: PropTypes.string,
```

Comment:
Dunno if you can see the test results https://circleci.com/gh/plotly/dash/24509, but they helpfully flagged this one: should be `config` with `update_title` nested inside a `PropTypes.shape`.

Reply:
I had a look at the implementation of the other core components and changed the line accordingly. I hope that is OK.